test/OptimisticTreasury.t.sol

```diff
@@ -67,8 +64,9 @@ contract OptimisticTreasuryTest is DSTest {
 		}
 	}
 
-	function testRetrieveEth(address _to) public {
-		if(_to == address(0x01) || address(this) == _to) return;
+	function testRetrieveEth() public {
+        address _to = address(0x11);
```

Comment:
Sorry, I don't understand this. Why is this line still there?
Line 69 is useless when you hardcode this address.

Reply:
yes I agree, this was fixed on a past commit, I haven't published yet